packages/eslint-config-base/package.json

```diff
@@ -7,7 +7,7 @@
     "prelint": "editorconfig-tools check * rules/* test/*",
     "lint": "eslint .",
     "tests-only": "babel-tape-runner ./test/test-*.js",
-    "prepublish": "(in-install || eslint-find-rules --unused) && (not-in-publish || npm test) && safe-publish-latest",
```

Comment:
curious why you'd remove this; this is very important for safety when doing backports.